src/profileReport.js

```diff
@@ -349,7 +383,7 @@ ProfileReport.prototype.renderIncomingBandwidth = function() {
 };
 
 ProfileReport.prototype.renderOperationSpeed = function(pureData, hasSecurity) {
-  var head = ["Path", "Count", "Average"];
+  var head = ["Path", "Count", "Average Execution Speed", "Pending Count", "Average Pending Time"];
```

Comment:
"Pending Count" may get confusing to our customers.

Reply:
I could omit that column altogther.